libs/net/dal/Services/ContentService.cs

```diff
@@ -179,7 +179,7 @@ public async Task<IPaged<Content>> FindWithElasticsearchAsync(ContentFilter filt
 
         if (!string.IsNullOrWhiteSpace(filter.OtherSource))
         {
-            filterQueries.Add(s => s.Term(t => t.OtherSource, filter.OtherSource.ToLower()));
+            filterQueries.Add(s => s.Term(t => t.SourceId, filter.OtherSource));
```

Comment:
This is incorrect, they don't mean the same thing.

Reply:
This is incorrect.  The UI can't use `otherSource` to filter for `sourceId`, they mean different things and are not even the same data type.  Please fix.

`otherSource` must filter the Elasticsearch `otherSource`
`sourceId` must filter the Elasticsearch `sourceId`

Reply:
Updated, thanks